src/main/kotlin/it/pagopa/ecommerce/payment/requests/client/NodeForPspClient.kt

```diff
@@ -33,14 +33,16 @@ class NodeForPspClient(
       .body(Mono.just(SoapEnvelope("", request)), SoapEnvelope::class.java)
       .retrieve()
       .onStatus(HttpStatus::isError) { clientResponse ->
-        clientResponse.bodyToMono(String::class.java).flatMap { errorResponseBody: String ->
-          Mono.error(ResponseStatusException(clientResponse.statusCode(), errorResponseBody))
-        }
+        Mono.error(
```

Comment:
the goal of the task is only related to the `checkPosition` and not to the `verify`